target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -218,6 +218,10 @@ qxwlan,e750a-v4-16m)
 	ucidef_set_led_switch "lan" "LAN" "green:lan" "switch0" "0x02"
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	;;
+glinet,gl-x300b|\
```

Comment:
So, not tested.

Reply:
I did test the firmware and noticed some issues with the lights, with the other changes they are now resolved.